src/redux/features/appVersion/appVersionSelectors.ts

```diff
@@ -0,0 +1,14 @@
+import { RootState } from 'src/redux/rootReducer';
+import { createSelector } from '@reduxjs/toolkit';
+
+export const selectAppVersion = (state: RootState) => state.appVersion.version;
+const currentAppVersion = process.env.REACT_APP_TERRAWARE_FE_BUILD_VERSION;
+
+export const selectIsAppVersionStale = (state: RootState) =>
+  !!state.appVersion.version && state.appVersion.version.toString().trim() !== currentAppVersion;
+
+// simple example of composable selectors
+export const selectIsAppVersionStale_EXAMPLE = createSelector(
```

Comment:
rename this to `selectIsAppVersionStaleExample` to pass the lint check.
I understand this is an example... we can check it in for future reference - at some point we will clean it up.